### PR TITLE
[Feat] Handle navigate in website with react-router-dom

### PR DESCRIPTION
**Pull request by an-nguyen-jh**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,5 +1,18 @@
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import Admin from "./pages/Admin/Admin";
+import Employee from "./pages/Employee/Employee";
+import Signin from "./pages/Signin/Signin";
+
```
Comment (anh-dinh-jh) [inline src/App.js]: Could we have a common file to export all pages from that file instead?

File: src/App.js [hunk 2]
```diff
@@ -1,5 +1,18 @@
...
 function App() {
-  return <></>;
+  return (
+    <BrowserRouter>
+      <Routes>
+        <Route exact path="/employee" element={<Employee></Employee>}></Route>
+        <Route exact path="/admin" element={<Admin></Admin>}></Route>
+        <Route exact path="/" element={<Signin></Signin>}></Route>
```
Comment (anh-dinh-jh) [inline src/App.js]: Is this prop `element` actually existing?

Comment (an-nguyen-jh) [reply src/App.js]: I am using react-router-dom v6. In this version, many props and components had been renamed. And in this case, component prop (v5) has been changed to element

Comment (anh-dinh-jh) [reply src/App.js]: ok

**Pull request merged**